### PR TITLE
fix: fix tag pages on blog/docs

### PR DESCRIPTION
**Pull request by DSchau**
This PR introduces the following fixes:

- Duplicate tags are largely eliminated on /blog/tags (e.g. React vs. react, JAMstack vs. jamstack, etc.)
- Only tags tied to blog posts are shown (e.g. fixes https://next.gatsbyjs.org/blog/tags/firebase/)
- Removes redundant gatsby tags (note: this was manual and not programmatic)

I feel like the toLowerCase() and kebabCase mix is a little janky. In the past, I've solved this by adding a custom slug resolver, but it's a bit harder to do with tags. That said, other approaches more than welcome :)

### CODE REVIEW

File: www/gatsby-node.js [hunk 1]
```diff
@@ -203,7 +203,7 @@ exports.createPages = ({ graphql, actions }) => {
 
       _.uniq(_.flatten(tagLists)).forEach(tag => {
         createPage({
-          path: `/blog/tags/${_.kebabCase(tag)}/`,
+          path: `/blog/tags/${_.kebabCase(tag.toLowerCase())}/`,
```
Comment (DSchau) [inline www/gatsby-node.js]: If we don't lowercase first, we get something like `ja-mstack` and `jamstack` as two separate groups. 🙃 https://runkit.com/dschau/5b8e9aea035ca50012829b75

**Comment by pieh** [comment]
I think this https://github.com/gatsbyjs/gatsby/blob/512de27947f020fc78181210a2e7798e905eac67/www/src/components/tags-section.js#L12 also need update (lowercase path before kebabcasing) (this is list of tags at the end of blog post page)

**Comment by DSchau** [comment]
@pieh as info addressed your comment!

**Comment by shannonbux** [comment]
Woo thanks *@DSchau!*

On Tue, Sep 4, 2018 at 12:47 PM, Michal Piechowiak <notifications@github.com

**Pull request merged**